c2corg_ui/static/js/documentediting.js

```diff
@@ -559,12 +548,24 @@ app.DocumentEditingController.prototype.preview = function() {
   var payload = {
     'document': angular.copy(this.scope[this.modelName])
   };
+  // TODO: loading indicator?
   this.http_.post(url, payload, config).
     catch(function(response) {
       this.alerts.addError('A server error prevented creating the preview');
     }.bind(this)).
     then(function(response) {
-      $('#preview-container-content').html(response['data']);
+      var template = angular.element('#preview-container').clone();
+      template.contents().filter(function() {
```

Comment:
I know :)
But the angular doc https://docs.angularjs.org/api/ng/function/angular.element says that 
> find() - Limited to lookups by tag name